src/scripting/Scripting.cpp

```diff
@@ -599,11 +609,45 @@ void Scripting::TriggerOnOverlayClose() const
     m_store.TriggerOnOverlayClose();
 }
 
+sol::object Scripting::GetModNames() const
+{
+    auto lockedState = m_lua.Lock();
+    sol::table res(lockedState.Get(), sol::create);
+    for (const auto& name : m_store.GetModNames())
+        res.add(name);
+    return res;
+}
+
 sol::object Scripting::GetMod(const std::string& acName) const
 {
     return m_store.GetMod(acName);
 }
 
+sol::object Scripting::GetModInputHandlers(const std::string& acName) const
+{
+    const auto* modBinds = m_store.GetBinds(acName);
+    if (!modBinds)
+        return sol::nil;
+
+    auto lockedState = m_lua.Lock();
+    const auto& bindings = m_store.GetBindings();
+    sol::table res(lockedState.Get(), sol::create);
+    for (const auto& bind : *modBinds)
+    {
+        sol::table info(lockedState.Get(), sol::create);
+        info["slug"] = bind.ID;
+        info["label"] = bind.DisplayName;
```

Comment:
Also would not unnecessarily call this something else, there was some plan for localization, it should be displayName, it is not really meant to be a label.

Reply:
You right, I'll rename it. I was just looking at names in wiki:
> -- registerInput()
> --
> -- @param  string    slug      The internal slug (must be unique in your mod scope)
> -- @param  string    label     The label displayed in CET Bindings
> -- @param  function  callback  The callback function

Reply:
wiki is community driven, we had not wrote that... ill notify somebody or make an edit later.